python/spec/fixtures/requirements/pip_compile_uv_header.txt

```diff
@@ -0,0 +1,22 @@
+# This file was autogenerated by uv via the following command:
+#    uv pip compile --output-file pip_compile_uv_header.txt unpinned_uv.in -P attrs==18.1.0
```

Comment:
The `-P` should not be added to the header. This looks like a bug in `uv`.

[`pip-compile` always excludes it](https://github.com/jazzband/pip-tools/blob/53309647980e2a4981db54c0033f98c61142de0b/piptools/utils.py#L56)

Reply:
Opened https://github.com/astral-sh/uv/issues/5031.

Reply:
Done, thanks @skshetry